### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by hpfmn** [danvd/wlroots-eglstreams#7] [General]
First of all thank you for making this possible. I also thaught it should be possible to also use eglstreams with wlroots but alas never had the time do try it myself. I'm glad that there are some pragmatic people who just do the things instead of always debating what would be the "best" solution ;)

Do you have any idea how to go forward with this project? Will you keep it as a fork as wlroots will progress (they already said, that they intend to make it even harder to use eglstreams)

**Comment by danvd** [comment]
While the principle of gpu back and primary buffers stays the same, I hope it won't be too hard to adopt current hacks to a possible new wlroots design. May be it sounds too optimistic, who knows.
